fuse/ipns/ipns_unix.go

```diff
@@ -285,7 +285,7 @@ func (s *Node) Attr() fuse.Attr {
 }
 
 // Lookup performs a lookup under this node.
-func (s *Node) Lookup(name string, intr fs.Intr) (fs.Node, fuse.Error) {
+func (s *Node) Lookup(name string, ctx context.Context) (fs.Node, error) {
```

Comment:
@tv42 can you shine a light on what is going on here? I don't understand how our updated code can confirm to the new interfaces where the `context` is the 1st argument.